{"uuid": "0cd00a00-7f1c-4a0f-8196-18adb252abeb", "children": ["22538fc5-8d02-4600-bbe0-f3c066ff46d9"], "befores": [{"name": "country", "status": "broken", "statusDetails": {"message": "AttributeError: module 'selenium.webdriver.chrome.webdriver' has no attribute 'Firefox'\n", "trace": "  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Python_Selenium\\Selenium\\Zeptoselenium\\Tests\\conftest.py\", line 109, in country\n    country = webdriver.Firefox(service=FirefoxService(GeckoDriverManager().install()))\n              ^^^^^^^^^^^^^^^^^\n"}, "start": 1730823698224, "stop": 1730823698224}], "start": 1730823698224, "stop": 1730823698412}